src/plugins/unified_search/public/__stories__/search_bar.stories.tsx

```diff
@@ -22,14 +22,15 @@ const mockIndexPatterns = [
     title: 'logstash-*',
     fields: [
       {
-        name: 'response',
+        name: 'bytes',
         type: 'number',
         esTypes: ['integer'],
         aggregatable: true,
         filterable: true,
         searchable: true,
       },
     ],
+    getName: () => 'logstash-*',
```

Comment:
ℹ️ This is irrelevant with the change, but the stories were breaking every time you clicked the add filter button. This change here solves the problem.